openshift/template.yaml

```diff
@@ -87,6 +87,12 @@ parameters:
   value: "false"
 - name: ENABLE_DELETE_UNREGISTER_GC
   value: "true"
+- name: DB_MAX_IDLE_CONNECTIONS
```

Comment:
you don't really need to set anything in the template if you already have the configuration in the code.

Reply:
it has another value cause we use rds here